<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="cell">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="create_cell">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать поле</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="size_n">
    <property name="geometry">
     <rect>
      <x>707</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="size_m">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>62</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>796</x>
      <y>62</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>m:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_unit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить юнита</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x">
    <property name="geometry">
     <rect>
      <x>707</x>
      <y>180</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>796</x>
      <y>182</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>182</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="y">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>176</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="wizard">
         <property name="text">
          <string>Wizard</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="panzer">
         <property name="text">
          <string>Panzer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="swordsman">
         <property name="text">
          <string>SwordsMan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="archer">
         <property name="text">
          <string>Archer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="centaur">
         <property name="text">
          <string>Centaur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="spearman">
         <property name="text">
          <string>SpearMan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>320</y>
      <width>171</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="move_unit">
       <property name="text">
        <string>Подвинуть юнита</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QSpinBox" name="diff_y">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="down">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSpinBox" name="diff_x">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="up">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="left">
         <property name="text">
          <string>Left</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="right">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
